packages/typo3-docs-theme/src/Settings/Typo3DocsThemeSettings.php

```diff
@@ -4,17 +4,31 @@
 
 class Typo3DocsThemeSettings
 {
+    /**
+     * @var array<string, string>
+     */
+    private static array $defaultSettings = [
```

Comment:
I suggest a backed enum for that.

Reply:
I didn't intend these as a finite list or enum. Just a way to define default values if certain settings are not set in the guides.xml.

Reply:
Maybe, but this list is finite. Or is there a way to hook into that array?